tuxemon/resources/db/locale/eo.json

```diff
@@ -5,18 +5,18 @@
     "combat_used_x_on_y": "${{name}} estis uzata al ${{target}} de ${{user}}!",
```

Comment:
You're right about using "kontraŭ" instead of "al" but not about ```{{name}}n``` as explained below. I'll make the correction